XenAdmin/TabPages/HAPage.cs

```diff
@@ -52,6 +52,12 @@ internal partial class HAPage : BaseTabPage
 
         private IXenObject xenObject;
 
+        private bool newPoolHABox;

```

Comment:
Also, the HBInitTimeElapsed and the timer could be renamed so that they have similat names (e.g. initializationDelayElapsed and initializationDelayTimer)